CM-Policy.md

```diff
@@ -24,4 +24,36 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
 
 For information on roles and responsibilities, management commitment, coordination among organizational entities, compliance, reviews, and updates please see the [Technology Transformation Service's (TTS) Common Control Policy](https://github.com/18F/compliance-docs/blob/master/TTS-Common-Control-Policy.md).
 
-## Procedures
\ No newline at end of file
+## Procedures
+
+cloud.gov's specific configuration management procedures are packaged with the actual code of the cloud.gov system. Below is an overview of our procedures along with citations to controls and relevant GitHub repositories.
+
+### 
+
+The cloud.gov team maintains a [Configuration Management Plan](https://github.com/18F/cg-docs/blob/master/content/ops/configuration-management.md) that governs configuration changes. The plan outlines the procedure for to make any and all changes impacting the configuration of the system
```

Comment:
I recommend linking to https://docs.cloud.gov/ops/configuration-management/ since that URL is more official (in the SSP statements I've helped with, I link to https://docs.cloud.gov/ops/configuration-management/ as the authoritative info + link to the GitHub page when explaining how we manage it).